### ISSUE TITLE
Close to Django alike feature

### ISSUE THREAD

**Issue by nikolas-dev** [tortoise/tortoise-orm#950]
I want more django alike feature. How it can be possible?
Creating object:
`Event.objects.create(name='Test')` instead of `Event.create(name='Test')`

Getting object:
`Event.objects.get(id=1)` instead of `Event.get(id=1)`

Filtering object:
`Event.objects.filter(name='Test') instead of Event.filter(name='Test')`

Deleting object:
`Event.objects.get(id=1).delete()` instead of `Event.get(id=1).delete()`

**Comment by nikolas310** [comment]
Yes I saw documentation. But custom manager has not create object
```
from tortoise.manager import Manager

class StatusManager(Manager):
    def get_queryset(self):
        return super(StatusManager, self).get_queryset().filter(status=1)


class ManagerModel(Model):
    status = fields.IntField(default=0)
    all_objects = Manager()

    class Meta:
        manager = StatusManager()
```
But when trying `Event.objects.create(name='Test')` is AttributeError: 'QuerySet' object has no attribute 'create'
